src/javascript/webgl-rendering-context.js

```diff
@@ -2281,6 +2305,8 @@ class WebGLRenderingContext extends NativeWebGLRenderingContext {
             case ext.MAX_COLOR_ATTACHMENTS_WEBGL:
               return super.getParameter(pname)
           }
+        } else if (this._extensions.oes_standard_derivatives && this._extensions.oes_standard_derivatives.FRAGMENT_SHADER_DERIVATIVE_HINT_OES === pname) {
```

Comment:
Should this be else-if? Can't `this._extensions.webgl_draw_buffers` and `his._extensions.oes_standard_derivatives` both be active?

Reply:
The conditions are more so testing `pname` against certain values, but only if the respective extension is enabled.

Reply:
Sorry if I'm slow heh, but it wont get to the getParemeter for the standard derivates if `this._extensions.webgl_draw_buffers` evaluates to true, right?

Reply:
OH! I see now. I believe you're right. I misread my own code.

I'll make the change. Thanks for catching that.